alpha/apps/kaltura/lib/myPartnerUtils.class.php

```diff
@@ -1486,6 +1486,10 @@ public static function copyUserRoles(Partner $fromPartner, Partner $toPartner)
  		UserRolePeer::setUseCriteriaFilter ( true );
  		foreach($roles as $role)
  		{
+ 			if ($role->getName() === 'cielo24')
```

Comment:
Need to have  param in local.ini
PartnerCopyUserIgnoreList='cielo24'

And future more users might be added. 
Get the users list from the kConf